hack/make/manifests/config.mk

```diff
@@ -1,4 +1,4 @@
-CRD_OPTIONS ?= "crd:crdVersions=v1"
```

Comment:
you are removing the description for all CRDs, even edgeconnect, and therefore making the generated API ref incorrect

is this really the best way to handle this problem?

Reply:
Dynakube descriptions are truncated only

Reply:
This will for now only affect dev. Before the next release we will take care that `v1beta3` can't be used if it's not ready and that descriptions of `v1beta2` will be fully available.